syncer/metrics/metrics.go

```diff
@@ -170,21 +197,22 @@ var (
 			Help:      "counter for syncer exits with error",
 		}, []string{"task", "source_id"})
 
-	ReplicationLagGauge = metricsproxy.NewGaugeVec(
-		prometheus.GaugeOpts{
+	ReplicationLagHistogram = metricsproxy.NewHistogramVec(
+		prometheus.HistogramOpts{
 			Namespace: "dm",
 			Subsystem: "syncer",
 			Name:      "replication_lag",
 			Help:      "replication lag in second between mysql and syncer",
-		}, []string{"task"})
+			Buckets:   prometheus.ExponentialBuckets(0.0000005, 2, 25), // this should be very fast.
```

Comment:
please find a proper range for replication lag (maybe exponential from 0.5s to 10m?) and notice the user that "Histogram metric is not an exact value"